cmd/captain/init.go

```diff
@@ -30,7 +30,6 @@ var mutuallyExclusiveParsers []parsing.Parser = []parsing.Parser{
 	new(parsing.JavaScriptMochaParser),
 	new(parsing.JavaScriptPlaywrightParser),
 	new(parsing.PythonPytestParser),
-	new(parsing.RubyCucumberParser),
```

Comment:
This is not mutually exclusive. Technically a breaking change, but I no one (other than us testing the integration) has used Ruby Cucumber. Checked if this is ok w/ Tommy

The outcome is you need to specify language and framework